tests/test_core.py

```diff
@@ -1617,17 +1577,15 @@ def test_column_type_schema(engine_testaccount):
 """
         )
 
-        table_reflected = Table(
-            table_name, MetaData(), autoload=True, autoload_with=conn
-        )
+        table_reflected = Table(table_name, MetaData(), autoload_with=conn)
         columns = table_reflected.columns
         assert (
             len(columns) == len(ischema_names_baseline) - 1
         )  # -1 because FIXED is not supported
 
 
 def test_result_type_and_value(engine_testaccount):
-    with engine_testaccount.connect() as conn:
+    with engine_testaccount.begin() as conn:
```

Comment:
Not starting a transaction explicitly was causing warnings. The code works in both versions still after this change.